### DISCUSSION TITLE
Package information from prior `tar_make`

### DISCUSSION THREAD

**Discussion by TylerGrantSmith** [ropensci/targets#704] [Help]
Is it possible to get the list of packages that were loaded from the meta data? (more specifically, the list of packages set via `tar_option_set(packages = ...)`?  It seems like `tar_option_get` only returns options/packages set in the current environment, and not from a previous run.

Thanks.

**Comment by wlandau** [comment]
`tar_renv()` crawls through the package settings and writes an R script with all the packages it thinks you will need to run the next `tar_make()` successfully (plus some extras for monitoring the pipeline). Otherwise, tracking that information is left to the user.

**Reply by TylerGrantSmith** [reply]
Ok, so for completeness, the options (defaults and set via `tar_option_set`)  that were used when running the pipeline are not persisted anywhere?

**Reply by wlandau** [reply]
Those options are not stored anywhere persistent, other than the code files you use to set them.
